website/docs/r/storage_account_network_rules.html.markdown

```diff
@@ -88,6 +88,17 @@ The following arguments are supported:
 
 -> **NOTE** User has to explicitly set `virtual_network_subnet_ids` to empty slice (`[]`) to remove it.
 
+* `resource_access_rules` - (Optional) A `resource_access_rules` block as defined below.
+
+---
+
+A `resource_access_rules` block supports the following:
```

Comment:
this seems to be related to private endpoints? so should this be named something different?